deploy/helm/eunomia-operator/values.yaml

```diff
@@ -34,17 +34,17 @@ eunomia:
 
     affinity: {}
 
-    resources: {}
+    resources:
       # We usually recommend not to specify default resources and to leave this as a conscious
       # choice for the user. This also increases chances charts run on environments with little
       # resources, such as Minikube. If you do want to specify resources, uncomment the following
       # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
-      # limits:
-      #   cpu: 100m
-      #   memory: 128Mi
-      # requests:
-      #   cpu: 100m
-      #   memory: 128Mi
+      limits:
+        cpu: 1000m
+        memory: 512Mi
+      requests:
+        cpu: 200m
+        memory: 256Mi
```

Comment:
Memory requests and limits must be the same.

Reply:
For starters let's try this ...

```
limits:
  cpu: 200m
  memory: 256Mi
requests:
  cpu: 200m
  memory: 256Mi
```